astro-iris/{{ cookiecutter.repo_name }}/src/setup.py

```diff
@@ -52,16 +52,13 @@
     install_requires=requires,
     extras_require={
         "docs": [
-            "sphinx>=1.6.3, <2.0",
-            "sphinx_rtd_theme==0.4.1",
-            "docutils",
-            "nbsphinx==0.3.4",
+            "sphinx~=3.4.3",
+            "sphinx_rtd_theme==0.5.1",
```

Comment:
```suggestion
            "sphinx_rtd_theme==0.5.1",
```

On kedro we have `0.4.1` - is it a typo here or are you just wanting to use latest versions?

Reply:
Hmm I copied this from Kedro. Let me double check.

Reply:
Yea I copied this from the main repo's starter: https://github.com/quantumblacklabs/private-kedro/blob/master/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/src/setup.py#L56